packages/server-wallet/src/wallet/__test__/integration/push-message.test.ts

```diff
@@ -67,7 +67,7 @@ it('stores states contained in the message, in a single channel model', async ()
   expect(channelsAfter[0].vars).toHaveLength(2);
 
   // The Channel model adds the state hash before persisting
-  expect(signedStates.map(addHash)).toMatchObject(channelsAfter[0].vars);
+  expect(channelsAfter[0].vars).toMatchObject(signedStates);
```

Comment:
Is the addHash not necessary?

Reply:
No, it was being used to mask an internal implementation detail